### ISSUE TITLE
Python error when skipping nondbshow search

### ISSUE THREAD

**Issue by poo706** [8cylinder/tv-overlord#34]
When doing a nondbshow search, if I don't want any of the resuts, I can [q]uit just fine, but if I [s]kip, I get the following:

```
Traceback (most recent call last):
  File "/usr/bin/tvol", line 11, in <module>
    sys.exit(tvol())
  File "/usr/lib/python3.5/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.5/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.5/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.5/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.5/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.5/site-packages/tvoverlord/tvol.py", line 375, in nondbshow
    nons.non_db(search_string, count)
  File "/usr/lib/python3.5/site-packages/tvoverlord/show.py", line 336, in non_db
    self._download(show_data)
  File "/usr/lib/python3.5/site-packages/tvoverlord/show.py", line 437, in _download
    search_type=Config.search_type)
  File "/usr/lib/python3.5/site-packages/tvoverlord/search.py", line 277, in download
    chosen_show, destination, final_name)
  File "/usr/lib/python3.5/site-packages/tvoverlord/search_providers/nzbclub_com.py", line 134, in download
    urllib.request.urlretrieve(href, fullname)
  File "/usr/lib/python3.5/urllib/request.py", line 188, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "/usr/lib/python3.5/urllib/request.py", line 163, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 451, in open
    req = Request(fullurl, data)
  File "/usr/lib/python3.5/urllib/request.py", line 269, in __init__
    self.full_url = url
  File "/usr/lib/python3.5/urllib/request.py", line 295, in full_url
    self._parse()
  File "/usr/lib/python3.5/urllib/request.py", line 324, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: 'skip'
```

**Comment by 8cylinder** [comment]
That has been fixed in the latest dev commit: 7fd3791afbf3f6d2ea1cbf2ed267d7a4a39dbdc1.
This bug: #32 also causes this one.

But this issue brings up a ui issue, this line:
`Letter, [s]kip, skip [r]est of show, [q]uit, [m]ark as downloaded, or [enter] for #1:`
should really only say:
`Letter or [q]uit`

The other options don't make sense when downloading a nondbshow.

**Comment by 8cylinder** [comment]
And while I'm thinking about it, `[enter] for #1:` should be removed from all options.  It seems redundant since the user can select a letter already.

**Comment by poo706** [comment]
Sounds like a good approach, I like it.

**Comment by 8cylinder** [comment]
fixed: 25fe99120eb3963fb8ed4a5f7a64037d35db1098
